Web UI/CognitiveSearch.UI/wwwroot/js/results.js

```diff
@@ -319,12 +323,17 @@ function UpdateResults(data) {
             name = "";
         }
 
+        var thumbnail_size_ext = "_300xN.jpg";
+        var thumbnail_path = "https://kmogstorage.blob.core.windows.net/kmdocument/THUMBNAILS/";
+        var filename = result.metadata_storage_name.split(".")[0];
+        var thumb_path = thumbnail_path + filename + thumbnail_size_ext + token;
+
         if (path !== null) {
             var classList = "results-div ";
             if (i === 0) classList += "results-sizer";
 
             var pathLower = path.toLowerCase();
-
+            //sets result for images
```

Comment:
Can you please explain the reason for these additional comments? lines 336, 348 365?
from initial look, they don't seem related to the thumbnail changes